Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/requirements.txt

```
@@ -0,0 +1,10 @@
+torch>=1.6.0
+mlflow>=1.12.0
+torchserve
+torch-model-archiver
+numpy
+transformers
+sklearn
+tqdm
+torchtext>=0.7.0
+pandas
```

```suggestion
pandas
google-cloud-storage
```
Not sure if you want that here or in a different spot

---

Actually sorry this doesn't make sense here.